portality/static/widget/fq_widget_depends_compiled.js

```diff
@@ -3269,372 +3269,3 @@ function journal_toc_id(journal) {
 
     return toc_id;
 }
-
```

Comment:
yeah, I discovered a weird difference between the various facetviews in doaj. public.journalaricle.facetview.js defines the rendering function before facetview declaration, whereas most of the others e.g  /admin.*\.js/ use rendering callbacks for the components of the result objects. 

the widget copied the public facetview, so I inherited the inferior workflow, and the closures below aren't required.